packages/block-renderer/src/components/pattern-renderer.tsx

```diff
@@ -1,13 +1,16 @@
+import usePatternInlineCss from '../hooks/use-pattern-inline-css';
 import usePatternMinHeightVh from '../hooks/use-pattern-min-height-vh';
 import BlockRendererContainer from './block-renderer-container';
 import { usePatternsRendererContext } from './patterns-renderer-context';
+
 interface Props {
 	patternId: string;
 	viewportWidth?: number;
 	viewportHeight?: number;
 	minHeight?: number;
 	maxHeight?: 'none' | number;
 	placeholder?: JSX.Element;
+	isNewSite: boolean;
```

Comment:
Since the block-renderer is a package that focuses on rendering blocks & patterns, it might be weird to know what's the new site. How about something like `shouldShufflePosts`?

Or we can generate the inline CSS from the parent and pass it to this component.

Reply:
I have renamed the prop to `shouldShufflePosts`. 

> Or we can generate the inline CSS from the parent and pass it to this component.

Let me know if you think that this other option is better. I'll be happy to refactor it. cc: @fushar